### PR TITLE
Polish - Mass - adding Holy Wednesday

### PR DESCRIPTION
**Pull request by mmolenda**
Problem in Holy Wednesday's Passion. It's similar to one described in issue #394. After 1955 changes, Passion starts with verse Luc 22:39, while Divinum Officum shows it from 22:1.
I made a fix to Latin and Polish propers.

### CODE REVIEW

File: web/www/missa/Latin/Tempora/Quad6-3.txt [hunk 1]
```diff
@@ -111,7 +115,10 @@ qui non habet, vendat túnicam suam, et emat gládium: Dico enim vobis,~
 quóniam adhuc hoc, quod scriptum est, oportet impléri in me: Et cum~
 iníquis deputátus est. Etenim ea, quæ sunt de me, finem habent. C. At~
 illi dixérunt: S. Dómine, ecce duo gládii hic. C. At ille dixit eis: J.~
-Satis est. C. Et egréssus ibat secúndum consuetúdinem in montem~
+Satis est. 
+(sed rubrica 1955 aut rubrica 1960 haec versus omittuntur)
+
+C. Et egréssus ibat secúndum consuetúdinem in montem~
```
Comment (gregordick) [inline web/www/missa/Latin/Tempora/Quad6-3.txt]: In 1955 onwards, this should be
> In illo tempore: Egressus Jesus ibat...

Comment (mmolenda) [inline web/www/missa/Latin/Tempora/Quad6-3.txt]: Good catch. Fixed.

File: web/www/missa/Latin/Tempora/Quad6-3.txt [hunk 2]
```diff
@@ -112,7 +116,12 @@ quóniam adhuc hoc, quod scriptum est, oportet impléri in me: Et cum~
 iníquis deputátus est. Etenim ea, quæ sunt de me, finem habent. C. At~
 illi dixérunt: S. Dómine, ecce duo gládii hic. C. At ille dixit eis: J.~
 Satis est. C. Et egréssus ibat secúndum consuetúdinem in montem~
-Olivárum. Secúti sunt autem illum et discípuli. Et cum pervenísset ad~
+Olivárum.
+(sed rubrica 1955 aut rubrica 1960 haec versus omittuntur)
+
+(rubrica 1955 aut rubrica 1960 dicitur)
+In illo témpore: Egréssus ibat secúndum consuetúdinem in montem Olivárum.~
```
Comment (gregordick) [inline web/www/missa/Latin/Tempora/Quad6-3.txt]: Still one more change, I'm afraid: 1955/60 have "Egressus *Jesus* ibat"

Comment (mmolenda) [inline web/www/missa/Latin/Tempora/Quad6-3.txt]: Sorry, fixed. Due to the wording nuances I would leave fixing this in other languages to respective authors.

**Comment by gregordick** [comment]
Thanks!  I'll open an issue so that the other languages don't get forgotten.

**Pull request merged**